bin/cmd.js

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env node
+
+import WikibaseRepo from './../lib/repo.js'
+import { pickLanguages, pickKeys } from './../lib/util.js'
+import { execCmd } from './../lib/exec.js'
+import { trap } from './../lib/signals.js'
+
+Promise.race([
```

Comment:
praise: I needed to quickly look up how this fancy thing worked and that's very cool. Looks suitably tidy!